docs/evo_abio.md

```diff
@@ -4,12 +4,12 @@ Q. Tyrell Davis and Josh Bongard
 
 
 {:style="text-align:center;"}
-[![Frog race](https://raw.githubusercontent.com/riveSunder/yuca/gecco_2022_pages/assets/halting_evo/frog_race.gif)](https://raw.githubusercontent.com/riveSunder/yuca/gecco_2022_pages/assets/halting_evo/frog_race.gif)
+[![Frog race](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/halting_evo/frog_race.gif)](https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/halting_evo/frog_race.gif)
 
-* [Blog post (you are here)](https://rivesunder.github.io/yuca/evo_abio)
+* [Blog post (you are here)](https://rivesunder.github.io/yuca_docs/evo_abio)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.